operator/build.gradle

```diff
@@ -138,7 +138,7 @@ task copyZetoSolidity(type: Copy) {
 }
 
 task buildContractSamples(type: Exec, dependsOn: [copySolidity, copyZetoSolidity]) {
-    commandLine "go", "run", "./contractpkg", "./contractpkg/contract_map.json"
+    commandLine "go", "run", "./contractpkg", "generate", "./contractpkg/contract_map.json"
```

Comment:
If possible, we should declare the inputs/outputs of these tasks so that Gradle knows when to re-run them (can be a follow-up).